policy/release/tasks.rego

```diff
@@ -197,11 +197,40 @@ current_required_tasks := task_data if {
 
 ## get the required task data for a pipeline with a label
 required_pipeline_task_data := task_data if {
-	some att in lib.pipelinerun_attestations
+	some att in _pipelinerun
 	count(tkn.tasks(att)) > 0
 	task_data := tkn.required_task_list(att)
 }
 
-_status(task) := status if {
+_status(task) = status if {
+	not task.status.conditions
```

Comment:
Consider reversing the order and adding a comment to explain the different use case:

```rego
_status(task) := status if {
  # Handle SLSA Provenance v1.0
  some condition in task.status.conditions
  condition.type == "Succeeded"
  status := _slsav1_status(condition)
} else := status if {
  # Handle SLSA Provenance v0.2
  status := task.status
} else := "MISSING"

Reply:
Interesting. If I switch the order and remove `not task.status.conditions`. The code doesn't work. I still need the `not`.

Reply:
completely rewritten in https://github.com/enterprise-contract/ec-policies/pull/709#discussion_r1333462257